examples/mitmproxy_shim.c

```diff
@@ -0,0 +1,79 @@
+#define _GNU_SOURCE
+#include <stdio.h>
+#include <string.h>
+#include <sys/prctl.h>
+#include <sys/types.h>
+#include <sys/capability.h>
+#include <unistd.h>
+#include <errno.h>
```

Comment:
A very short comment what this does would be nice here (also: a more descriptive file name?)